clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -723,7 +737,15 @@ struct MCDCCoverageBuilder {
     if (I == CondIDs.end())
       return -1;
     else
-      return I->second;
+      return I->second.ID;
+  }
+
+  void ccc(const Expr *CondExpr, mcdc::ConditionIDs IDs) {
```

Comment:
this should be named differently, based on name I don't understand what it is doing

Reply:
This is w-i-p. Thanks for pointing this out.